File: src/test/java/name/seguri/java/leetcode/_242/SolutionTest.java
package name.seguri.java.leetcode._242;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class SolutionTest {
  private final Solution sut = new Solution();

  @Test
  void input1() {
    var result = sut.isAnagram("anagram", "nagaram");

    assertThat(result).isTrue();
  }

  @Test
  void input2() {
    var result = sut.isAnagram("rat", "car");

    assertThat(result).isFalse();
  }
}

File: src/test/kotlin/name/seguri/kotlin/leetcode/_242/SolutionTest.kt
package name.seguri.kotlin.leetcode._242

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class SolutionTest {
  private val sut = Solution()

  @Test
  fun input1() {
    val result = sut.isAnagram("anagram", "nagaram")
    assertThat(result).isTrue()
  }

  @Test
  fun input2() {
    val result = sut.isAnagram("rat", "car")
    assertThat(result).isFalse()
  }
}
